global.json

```diff
@@ -1,18 +1,18 @@
 {
   "sdk": {
-    "version": "6.0.100-rc.2.21505.57",
+    "version": "6.0.100",
     "allowPrerelease": true,
     "rollForward": "disable"
   },
   "tools": {
-    "dotnet": "6.0.100-rc.2.21505.57",
+    "dotnet": "6.0.100",
     "vs": {
       "version": "16.10"
     },
     "xcopy-msbuild": "16.10.0-preview2"
   },
   "msbuild-sdks": {
-    "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.21474.2",
-    "Microsoft.DotNet.Helix.Sdk": "7.0.0-beta.21474.2"
+    "Microsoft.DotNet.Arcade.Sdk": "7.0.0-beta.21566.10",
+    "Microsoft.DotNet.Helix.Sdk": "7.0.0-beta.21566.10"
   }
-}
+}
```

Comment:
⚠️ This should not have changed. If a tool created this change, please file a bug report for that tool (changing the line ending at the end of the file is not allowed).

Reply:
Looks like it happened when I resolved conflicts locally. Should we update our .editorconfig to specify what we want for .json files.

Reply:
```suggestion
}

```